source/geometries/Next100FieldCage.cc

```diff
@@ -155,6 +158,16 @@ Next100FieldCage::Next100FieldCage(G4double grid_thickn):
   ELlong_diff_cmd.SetParameterName("ELlong_diff", true);
   ELlong_diff_cmd.SetUnitCategory("Diffusion");
 
+  G4GenericMessenger::Command&  drift_vel_cmd =
+  msg_->DeclareProperty("drift_v", drift_v_,
+                        "The active volume drift velocity");
+  drift_vel_cmd.SetParameterName("drift_v", true);
+
+  G4GenericMessenger::Command&  EL_drift_vel_cmd =
+  msg_->DeclareProperty("EL_drift_v", EL_drift_v_,
+                        "The EL region drift velocity");
+  EL_drift_vel_cmd.SetParameterName("EL_drift_v", true);
+
```

Comment:
Can you add the restriction that they are >= 0?

Reply:
just added now